pinot-query-runtime/src/test/resources/queries/QueryHints.json

```diff
@@ -83,6 +83,8 @@
       },
       {
         "description": "Colocated JOIN with partition column with partition parallelism in first table",
+        "ignored": true,
+        "comment": "partition parallelism mismatched in hint, this query shouldn't work at all",
```

Comment:
previously we decided to allow this but i guess we should not. putting an ignore here first but i think we should not allow this and should throw exception